Unix/samples/Providers/Test_Indication/fire.c

```diff
@@ -161,7 +161,7 @@ PAL_Uint32 THREAD_API fireindication(void* param)
 
     if (MI_FALSE == config->disabled)
     {
-        LOGMSG(("fireindication stopped due to class (%s) need to fail after firing (%d) indicaitons", config->className, failAfterCount));
+        LOGMSG(("fireindication stopped due to class (%s) need to fail after firing (%d) indications", config->className, failAfterCount));
```

Comment:
I'd love to replace `fireindication`, if only with `fireIndication` or `FireIndication`